Slovenian mode

janezpavelzebovec opened the pull request:
The mode is updated in the following sections:
- The V and J after the vowel and before the consonant are pronounced as U and I and together with the preceding vowel form a diphthong;
- Assimilation in consonant clusters, where the preceding consonant adopts the voicing of the consonant behind it;
- Marks for homoorganic nasalized consonants;
- Shorter list of words with special/unpredictable pronunciation
- Updated transcription for diphthongs, as some were missing.

arnog commented:
Thanks for your contribution with this PR. I have a few questions:
- did you attempt to contact the author of the previous version of the Slovenian mode to get their input on your proposed changes?
- I would suggest keeping the name of the mode file to "slovenian-orthographic". As far as I can tell, it is still an orthographic mode, but by renaming it it makes comparison the previous version more difficult.
- Could you describe some example of what problems these changes address? Is it just a question of style/personal preference, or where there objectively incorrect transcriptions with the previous version?
- It seems there are more rules than necessary. For example, there is a pre-process rule for: "ə": "e". As a result "ə" will never appear in the left-hand side of the map rules, yet a number of those rules include it. There also the introduction of a "ŋ" in the preprocess rule, which doesn't seem necessary. It seems like three rules that map "ng", "nk" and "nh" to nwalme would be clearer.

janezpavelzebovec commented:
Hello!

1. No, actually I didn't attempt to contact him, because my corrections are not a matter of adapting the Tengwar script, but rather of phonological characteristics. However, I can contact him if you wish.
2. I removed the orthographic tag from the file name because this Slovenian mode is based on phonological features, not orthographic ones. In addition, most modes in other languages do not have an orthographic tag, although some are orthographic. But, of course, the filename has no visible effect, so it doesn't really matter if the name stays as it is.
3. The mode is updated in the following sections:
- The V and J after the vowel and before the consonant are pronounced as U and I and together with the preceding vowel form a diphthong;
- Assimilation in consonant clusters, where the preceding consonant adopts the voicing of the consonant behind it;
- Marks for homoorganic nasalized consonants;
- Shorter list of words with special/unpredictable pronunciation
- Updated transcription for diphthongs, as some were missing.
4. And for ə and ŋ you are right. I will fix it.

janezpavelzebovec commented:
Hello!

I have contacted the author of the previous version (_u/Zellas06_). To summarise, his reply is mainly that this is an orthographic, not a phonemic mode; so the changes (in general) don't really make sense.
I myself somehow took this version as an unfinished phonemic version, as it contains some phonemic elements: diphthongs (to the extent that they are written), and palatalized N and L. This was probably also due to the belief that Tengwar is a completely different/foreign script, by someone who does not write our language in our script (in his world, our script does not even exist - although neither does our language :) ); consequently, when writing our (spoken) language in Tengwar, we should not look how is it written in our script. It probably didn't help that I obviously didn't understand exactly what the correct orthographic mode is supposed to be.
For all this, I sincerely apologise.

Now I wonder if there should be only one mode for each language in Tecendil (except English, of course), or can there be both orthographic and phonemic? In that case I would make a new pull request, keeping some improvements for the orthographic mode, and creating a new Slovenian phonemic one.
